### PR TITLE
Fix UG errors from PED

### LINKED ISSUE #120: [PE-D] Documentation issue for Command Summary redirect
Bug Name: Redirection issue for Command Summary:

Bug id: 1

Area Path: User Guide -> Command Summary

Severity: Low

Description: Command Summary redirect does not redirect to them bottom of the command summary part. Instead, it seems to redirect to itself.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: HangZelin/ped#1

### LINKED ISSUE #125: [PE-D] Inconsistency in UG for help command
Bug Name: In consistency in UG for `help` command.

Bug id:6

Area Path: User Guide -> Features Note

Severity: Low

Description: In UG, it is stated that `help` command does not take in any parameters but it does. Also, `help 123` should be viewed as `help` but in the actual CONNECTION, it is viewed as a error command input.

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: HangZelin/ped#7

### LINKED ISSUE #129: [PE-D] UG: repeated titles
-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: fans2619/ped#1

### LINKED ISSUE #131: [PE-D] UG: empty body with a "Command Summary" header there
-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: fans2619/ped#2

### LINKED ISSUE #136: [PE-D] Error in UG Removing Tags
First bullet point incorrect states that `untag` adds a tag to a person

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: nujiak/ped#3

### LINKED ISSUE #149: [PE-D] UG: wrong information given
-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: fans2619/ped#8

### LINKED ISSUE #153: [PE-D] phone numbers can be of any length
To recreate this issue, use add command to add a person and after the p/ prefix, you can add any number of digits. Command will still work.
The output can be seen below:

Perhaps you can set the limit to 8 digits as that is the standard length of Singaporean phone numbers.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: MGRL2201/ped#8

### LINKED ISSUE #154: [PE-D] Should update outdated information in User Guide for Help command
In the section "Notes about the command format" in the UG: 

States that "help 123" will be interpreted as "help",

but an error message is displayed. 

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: hanif-kamal/ped#5

### LINKED ISSUE #165: [PE-D] Inconsistent header size in User Guide
The "Format:" text for unpin and delete commands are of different size. Would be good to standardize the size in Markdown.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: hanif-kamal/ped#9

### LINKED ISSUE #166: [PE-D] Hyperlinks for mailingList and findAny commands in Table of Contents don't work
Tried clicking on hyperlinks for mailingList and findAny commands, but nothing happened. Rest of the hyperlinks work fine (except Command Summary, but that's a separate issue). 

Might need to check the Markdown code again for the hyperlinks.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: hanif-kamal/ped#10

### LINKED ISSUE #167: [PE-D] Formatting for "Command Summary" in Table of Contents is inconsistent + hyperlink doesn't work
The font size for "Command Summary" is way huger than the rest of the bullet points in the table of contents. 

Also, clicking on the hyperlink just moves the page down slightly, instead of jumping down to the Command Summary section at the bottom of the document.

Might need to check the Markdown code to fix.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: hanif-kamal/ped#11

### LINKED ISSUE #168: [PE-D] User Guide header appears twice on the website
User Guide header appears twice on the website.

Should be very simple to fix in Markdown.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: hanif-kamal/ped#12

### LINKED ISSUE #169: [PE-D] Some sentences in User Guide don't have fullstops at the end
Some examples: 

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: hanif-kamal/ped#13

### PR DESCRIPTION
**Pull request by Jiwei99**
Fixes #169, fixes #168, fixes #167, fixes #166, fixes #165, fixes #154, fixes #153, fixes #149, fixes #136, fixes #131, fixes #129, fixes #125, fixes #120

### CODE REVIEW

**Review by yucheng11122017 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ Removes an existing tag from an existing person in the address book.
 * `untag INDEX [t/TAG]…​`
```
Comment (leezhixuan) [inline docs/UserGuide.md:179]: Should we include a note saying that at least 1 tag must be provided? `untag INDEX [t/TAG]` suggests that the `t/TAG` is optional

**Review by leezhixuan — APPROVED** [review]
1 point to consider, otherwise LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (Jiwei99) [reply docs/UserGuide.md]: Updated it to `untag INDEX t/TAG [t/MORE_TAGS]…​`

**Pull request merged**